lib/fpm/package/deb.rb

```diff
@@ -905,7 +905,7 @@ def mkdir_p(dir)
   def to_s(format=nil)
     # Default format if nil
     # git_1.7.9.3-1_amd64.deb
-    return super("NAME_FULLVERSION_ARCH.TYPE") if format.nil?
+    return super("NAME_FULLVERSION_ARCH.EXTENSION") if format.nil?
```

Comment:
haha, I agree that this is more than a bit awkward. It certainly looks like one of those "code smell" type of problems where it just feels weird. We could rewrite it like this, using default argument:

```
def to_s(format="NAME_FULLVERSION_ARCH.EXTENSION")
  super(format)
end
```

And then I wonder why this method exists just to have a default value in a weird way that only really calls super(...). We could probably refactor this to have each package type be able to describe it's own default format, then not need to define `to_s` for each package type.

A refactor for another day, perhaps ;P

(If you dive into doing this, I support it, happy to review whatever)